changelog

```diff
@@ -1,3 +1,142 @@
+Version 0.10.9
+  * Big changes:
+    * Added an alien faction dwelling in a gas giant in the galactic Northeast. (@RisingLeaf, @GefullteTaubenbrust2)
```

Comment:
I would like to include roadrunner here for exceptional writing help, a lot of people helped me, but roadrunner basically wrote over 50% of the mission text

Reply:
```suggestion
    * Added an alien faction dwelling in a gas giant in the galactic Northeast. (@RisingLeaf, @GefullteTaubenbrust2, @roadrunner56)
```
Btw, should these be in alphabetical order?